components/console/helpers/questionhelper.rst

```diff
@@ -234,6 +234,36 @@ You can also specify if you want to not trim the answer by setting it directly w
         $name = $helper->ask($input, $output, $question);
     }
 
+Accept Multiline Answers
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.2
+
+    The ``setMultiline()`` and ``isMultiline()`` methods were introduced in
+    Symfony 5.2.
+
+By default, the question helper stops reading user input when it receives a newline
+character (i.e., when the user hits ``ENTER`` once). However, you may specify that
+the response to a question should allow multiline answers by passing ``true`` to
+:method:`Symfony\\Component\\Console\\Question\\Question::setMultiline`::
+
```

Comment:
i would just add somethin like

```php
// src/Command/MyCommand.php
```

Reply:
I'm not sure what you mean.

Reply:
Never mind. I see what you mean now. The context for your comment didn't show the lines below, so I thought you were talking about the lines above. 😁 

None of the other examples on this page show an example name for their command script. If I add this here, should I also add it to the rest of the examples?

Reply:
ho i did not figure that other have not
but yes i see a lot on code example the path file on top as a comment
it helps understanding in which file the class is and its meaning etc :) 

but i do not know if you should update all others :s

Reply:
I don't disagree, but it's inconsistent with the rest of the examples in this file.